### PR TITLE
mpv: update to 0.30.0.

### PR DESCRIPTION
**Pull request by pbui**
- Updated libplacebo to 1.21.0
- Bumped vlc (due to libplacebo)

- Changes: https://github.com/mpv-player/mpv/releases/tag/v0.30.0
- Tested with audio and video (x86_64)

### CODE REVIEW

**Comment by xtraeme** [comment]
You need to update `common/shlibs` for `libplacebo-1.21.0`

File: srcpkgs/mpv/template [hunk 1]
```diff
@@ -1,47 +1,47 @@
 # Template file for 'mpv'
 pkgname=mpv
-version=0.29.1
-revision=7
+version=0.30.0
+revision=1
 build_style=waf
 configure_args="--confdir=/etc/mpv --docdir=/usr/share/examples/mpv
- --enable-dvdread --enable-dvdnav --enable-cdda --enable-libmpv-shared
+ --enable-dvdnav --enable-dvdnav --enable-cdda --enable-libmpv-shared
```
Comment (zdtcd) [inline srcpkgs/mpv/template]: duplicate value of `--enable-dvdnav`

File: srcpkgs/mpv/template [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
  $(vopt_enable pulseaudio pulse) $(vopt_enable sdl2)
- $(vopt_enable smb libsmbclient) $(vopt_enable sndio) $(vopt_enable v4l2 tv)
+ $(vopt_enable smb libsmbclient) $(vopt_enable sndio)
  $(vopt_enable vapoursynth) $(vopt_enable vdpau) $(vopt_enable wayland)
  $(vopt_enable x11) $(vopt_enable vulkan) $(vopt_enable vulkan shaderc)"
 hostmakedepends="pkg-config python-docutils perl $(vopt_if wayland wayland-devel)"
 makedepends="MesaLib-devel ffmpeg-devel harfbuzz-devel lcms2-devel libXv-devel
  libass-devel libbluray-devel libcdio-paranoia-devel libdvdnav-devel
- libdvdread-devel libguess-devel libuuid-devel libva-glx-devel rubberband-devel
+ libdvdnav-devel libguess-devel libuuid-devel libva-glx-devel rubberband-devel
```
Comment (zdtcd) [inline srcpkgs/mpv/template]: duplicate value of `libdvdnav-devel`

Comment (xtraeme) [reply srcpkgs/mpv/template]: good catch :+1:

Comment (xtraeme) [reply srcpkgs/mpv/template]: If we'd change this to proper bash arrays we could detect them easily...

Comment (xtraeme) [reply srcpkgs/mpv/template]: and at a reasonable speed...

**Comment by xtraeme** [comment]
Already applied... sorry did this via cmdline

File: srcpkgs/mpv/template (hunk 2 above)
Comment (pbui) [reply srcpkgs/mpv/template]: Ah, thanks for the review... I actually did update the common/shlibs, but I forgot to include it in the  commit.  Thanks for cleaning up and merging the PR.

**Pull request merged**